src/object_identitifer.rs

```diff
@@ -119,4 +142,21 @@ mod tests {
             assert!(ObjectIdentifier::from_string(val).is_some());
         }
     }
+
+    #[test]
+    fn test_to_string() {
+        for val in &[
+            "2.5",
+            "2.5.2",
+            "1.2.840.113549",
+            "1.2.3.4",
+            "1.2.840.133549.1.1.5",
+            "2.100.3",
```

Comment:
One last test request: something from the < 1 arc (e.g. 0.4.whatever)

Reply:
Done.